lib/fathead/mdn_css/output.txt

```diff
@@ -344,28 +389,151 @@
+<border-style>	A										<section class="prog__container"><p>See border-style</p></section>	https://developer.mozilla.org/en-US/docs/Web/CSS/border-inline-start#Values
+<border-width>	A										<section class="prog__container"><p>See border-width</p></section>	https://developer.mozilla.org/en-US/docs/Web/CSS/border-block-start-width#Values
+<box>	A										<section class="prog__container"><p>See background-clip</p></section>	https://developer.mozilla.org/en-US/docs/Web/CSS/background#Values
+<br-style>	A										<section class="prog__container"><p>Is a keyword describing the style of the bottom border. It can have the following values:\n \n  \n   \n    none\n    \n      \n    \n    Like for the hidden keyword, displays no border. In that case, except if a background image is set, the calculated values of border-width will be 0, even if specified otherwise through the property. In case of table cell and border collapsing, the none value has the lowest priority: it means that if any other conflicting border is set, it will be displayed.\n   \n   \n    hidden\n    \n      \n    \n    Like for the none keyword, displays no border. In that case, except if a background image is set, the calculated values of border-width will be 0, even if specified otherwise through the property. In case of table cell and border collapsing, the hidden value has the highest priority: it means that if any other conflicting border is set, it won't be displayed.\n   \n   \n    dotted\n    \n      \n    \n    Displays a series of rounded dots. The spacing of the dots is not defined by the specification and is implementation-specific. The radius of the dots is half the calculated border-width.\n   \n   \n    dashed\n    \n      \n    \n    Displays a series of short square-ended dashes or line segments. The exact size and length of the segments are not defined by the specification and are implementation-specific.\n   \n   \n    solid\n    \n      \n    \n    Displays a single, straight, solid line.\n   \n   \n    double\n    \n      \n    \n    Displays two straight lines that add up to the pixel amount defined as border-width .\n   \n   \n    groove\n    \n      \n    \n    Displays a border leading to a carved effect. It is the opposite of ridge.\n   \n   \n    ridge\n    \n      \n    \n    Displays a border with a 3D effect, like if it is coming out of the page. It is the opposite of groove.\n   \n   \n    inset\n    \n      \n    \n    Displays a border that makes the box appear embedded. It is the opposite of outset. When applied to a table cell with border-collapse set to collapsed, this value behaves like groove.\n   \n   \n    outset\n    \n      \n    \n    \n     Displays a border that makes the box appear in 3D, embossed. It is the opposite of inset. When applied to a table cell with border-collapse set to collapsed, this value behaves like ridge.</p></section>	https://developer.mozilla.org/en-US/docs/Web/CSS/border-style#Values
```

Comment:
i think we're going to want to normalize multiple `\n` to single or double newlines? This looks like its got way too many